numba/cuda/stubs.py

```diff
@@ -308,5 +308,25 @@ class max(Stub):
         differs from Python and Numpy behaviour, where max(a, b) is always
         a when either a or b is a NaN.
 
-        Supported on float64 operands only.
+        Supported on int32, int64, uint32, uint64, float32, float64 operands only.
+        """
+
+    class min(Stub):
+        """min(ary, idx, val)
```

Comment:
The documentation here fails to mention the return value (same for `cuda.atomic.max()`).